packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -217,7 +221,8 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   ///
   /// Becomes the leading component of the [NavigationToolBar] built
   /// by this widget. The [leading] widget's width and height are constrained to
-  /// be no bigger than toolbar's height, which is [kToolbarHeight].
+  /// be no bigger than default toolbar's height, which is [kToolbarHeight] and
+  /// [toolbarHeight] respectively.
```

Comment:
I'm not sure I follow this comment. Which will it be constrained to, kToolbarHeight or toolbarHeight?

Reply:
I'll improve it in the next commit.